llama-index-integrations/vector_stores/llama-index-vector-stores-azureaisearch/llama_index/vector_stores/azureaisearch/base.py

```diff
@@ -497,6 +508,7 @@ def __init__(
                 Dict[str, Tuple[str, MetadataIndexFieldType]],
             ]
         ] = None,
+        hidden_field_keys: List[str] = [],
```

Comment:
default arguments should never be a list or dict due to how pass-by-reference works in python

Reply:
Pushed a fix